db/migrate/20150312082018_create_videos.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVideos < ActiveRecord::Migration
+  def change
+    create_table :videos do |t|
+      t.string :video_id
+      t.string :amara_link
```

Comment:
The Amara link can be generated dynamically using the YouTube link. We want to store the YouTube link instead.